packages/bump-monorepo-packages/src/get-packages-in-topological-order.js

```diff
@@ -0,0 +1,62 @@
+const { promises: fs } = require('fs');
+const path = require('path');
+const { glob } = require('glob');
+const toposort = require('toposort');
+
+module.exports = async function getPackagesInTopologicalOrder(cwd) {
```

Comment:
This seems very useful for a generic `lerna` replacement where we use it to run scripts in topological order :+1: